data/encounters/encounter.json

```diff
@@ -0,0 +1,9 @@
+{
+  "encounter":  {
+    "name": "Goblin Attack!",
+    "monsters": [
+      {"race": "goblin", "max_health": "8", "name": "Goblin 1"},
```

Comment:
The convention is to have an empty space inside of curly brackets.

It's not important, but would be a nice change. :+1: